src/eterna/Plot.ts

```diff
@@ -222,8 +222,8 @@ export default class Plot extends Container {
     private _upperBounds: number[] | null;
     private _labelFields: Text[] | null;
 
-    private _width: number = 100;
-    private _height: number = 100;
+    protected _width: number = 100;
```

Comment:
Is this because classes extending Plot need its w/h or because Container has _width and _height and you want to override them?

Reply:
The latter. I could be convinced that we should do something different, but by the fact they're protected, I imagine this approach is more or less intended?